src/translations/pages/assistant.ts

```diff
@@ -38,7 +39,61 @@ export const Assistant = {
   },
   trip: {
     tripPattern: {
-      busFrom: _('Buss fra', 'Bus from', 'Buss frå'),
+      travelFrom: (mode: TransportModeType, place: string) => {
+        switch (mode) {
+          case 'bus':
+          case 'coach':
+            return _(
+              `Buss fra ${place}`,
+              `Bus from ${place}`,
+              `Buss frå ${place}`,
+            );
+          case 'tram':
+            return _(
+              `Trikk fra ${place}`,
+              `Tram from ${place}`,
+              `Trikk frå ${place}`,
+            );
+          case 'metro':
+            return _(
+              `T-bane fra ${place}`,
+              `Metro from ${place}`,
+              `T-bane frå ${place}`,
+            );
+          case 'rail':
+            return _(
+              `Tog fra ${place}`,
+              `Train from ${place}`,
+              `Tog frå ${place}`,
+            );
+          case 'water':
+            return _(
+              `Båt fra ${place}`,
+              `Boat from ${place}`,
+              `Båt frå ${place}`,
+            );
+          case 'air':
+            return _(
+              `Fly fra ${place}`,
+              `Air from ${place}`,
+              `Fly frå ${place}`,
+            );
+          case 'bicycle':
+            return _(
+              `Sykkel fra ${place}`,
+              `Bike from ${place}`,
+              `Sykkel frå ${place}`,
+            );
+          case 'foot':
+            return _(
+              `Gå fra ${place}`,
+              `Walk from ${place}`,
+              `Gå frå ${place}`,
+            );
```

Comment:
I think the point of this header should be the first thing you have to look for. So in the app that is things like stop places. I don't think "walk from" make sense as you in most cases know where you are or what you have searched for. So I think the logic should be "find first leg that has a quay".

But the specification of transport mode I think is good!

Reply:
Yes, thought it looked weird with "Walk from"